scripts/prepare-test-ledger.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# Usage: ./scripts/prepare-test-ledger.sh <BP key 1> <BP key 2> ... <BP key n>
+
+# Number of keys in ledger that won't be re-delegated
+NUM_GHOST_KEYS=${NUM_GHOST_KEYS:-0}
+KEY_BALANCE=100000
+
+echo "Script assumes mainnet's start was at epoch 0 on 17th March 2021, if it's not the case, update the script please" >&2
+
+if [[ $# -eq 0 ]]; then
+  echo "No arguments specified: need block producing keys" >&2
```

Comment:
NIT:

Maybe just write content of line 3: 
`Usage: ./scripts/prepare-test-ledger.sh <BP key 1> <BP key 2> ... <BP key n>`

It is more clear how to user scripts than  `block producing keys`